settings/security/third-party/gitlab.rst

```diff
@@ -16,15 +16,16 @@ where zammad_host has to be replaced with your Zammad FQDN
    :alt: Register OAuth app on gitlab instance
 
 At the moment we need the "api" scope. This is caused due a bug
-in Gitlab: https://gitlab.com/gitlab-org/gitlab-ce/issues/33022
+in Gitlab.
```

Comment:
I think we should not touch this and I will create a follow-up fix, with this no API-Scope in general is no longer needed.